### ISSUE TITLE
The government stripped the licenses of more than 1,600 NGOs accused of economic  fraud and corruption.

### ISSUE THREAD

**Issue by BKP05** [colaresi/AnnotationProjectPULSAR#19]
```
The government stripped the licenses of more than 1,600 NGOs accused of economic 
fraud and corruption.
```
I guess that the aspect is __NGOs accused of economic fraud and corruption__. 
__more than 1,600__ is the size of the action, so it would be appropriate to be put together with the judgment category?

&rightarrow; aspect: *NGOs accused of economic fraud and corruption* 
&rightarrow; judgment: *stripped the licenses of more than 1,600*
&rightarrow; perpetrator: *the government* 
&rightarrow; victim: *NGOs*

**Comment by colaresi** [comment]
I agree here if we interpret that government as doing this to block NGOS. Then the point would be that treatment of NGOs is a known aspect of human rights, and here economic fraud and corruption are not accessions about the government but what the government is accusing the NGOs of.

On the other hand, if the NGOs really did do these things, then the aspect is the economic fraud and corruption.  The context of the paragraph should make this clear. Does it?

**Comment by BKP05** [comment]
That's right. That's why it is important to check where the sentence comes from.  It is from "Section 4 Governmental Attitude Regarding International and Nongovernmental Investigation of Alleged Violations of Human Rights".  The context is that a government is trying to shut down NGO activities.  So, it would be reasonable to see this as the treatment of NGOs, not economic fraud and corruption.

**Comment by colaresi** [comment]
Agreed.

**Comment by colaresi** [comment]
PULSAR 3 Beta coding

```
<Perpetrator>The government</Perpetrator> <SAVP, attribute="violation">stripped the licenses of</SAVP> <JQu, attribute="high">more than 1,600</JQu> <GAHR>NGOs</GAHR> <TaTVP>accused of economic 
fraud and corruption </TaTVP>.
```
